probe-rs/src/architecture/arm/mod.rs

```diff
@@ -28,26 +28,21 @@ pub use swo::{SwoAccess, SwoConfig, SwoMode, SwoReader};
 pub use traits::*;
 
 /// ARM-specific errors
-#[derive(Debug, thiserror::Error)]
+#[derive(Debug, thiserror::Error, docsplay::Display)]
 pub enum ArmError {
-    /// The operation requires a specific architecture.
-    #[error("The operation requires one of the following architectures: {0:?}.")]
+    /// The operation requires one of the following architectures: {0:?}.
     ArchitectureRequired(&'static [&'static str]),
 
     /// A timeout occurred during an operation
```

Comment:
```suggestion
    /// A timeout occurred during an operation.
```

Reply:
Should I also add a `.` after the other `{0}` ?

Reply:
no, I think that would introduce double dots in some cases.